src/data/biomes/island.ts

```diff
@@ -1,12 +1,29 @@
 import { Biome } from "#app/data/biome";
-import { townPokemonPool, townTrainerPool, townWeatherPool, townTerrainPool } from "#app/data/biomes/town";
+import { townPokemonPool, townTrainerPool, townTerrainPool } from "#app/data/biomes/town";
 import { BiomeId } from "#enums/biome-id";
+import { WeatherType } from "#enums/weather-type";
 
+/**
+ * 1/8 for rain, 1/4 for sun if dawn/day
```

Comment:
NIT: Keep denominators the same for ease of reading:
```suggestion
 * 1/8 for rain, 2/8 for sun if dawn/day
```